docs/source/models/vlm.rst

```diff
@@ -150,7 +150,7 @@ A full code example can be found in `examples/openai_vision_api_client.py <https
 
     .. code-block:: shell
 
-        export VLLM_IMAGE_FETCH_TIMEOUT=<timeout>
+        export VLLM_MULTIMODAL_FETCH_TIMEOUT=<timeout>
```

Comment:
Given that different types of data may have different sizes (think image vs video), it may be better to have a separate timeout for each modality.